teachertool/src/services/storageService.ts

```diff
@@ -152,7 +152,7 @@ export async function getRubric(name: string): Promise<Rubric | undefined> {
     const db = await getDb;
 
     let rubric: Rubric | undefined = undefined;
-    if (name) {
+    if (name !== undefined) {
```

Comment:
I think this conditional is unnecessary because `name` cannot be undefined. Maybe just remove it entirely?